.test/config/config.yaml

```diff
@@ -43,6 +43,13 @@ diffexp:
     ma-plot: 0.05
     qq-plot: 0.05
 
+diffsplice:
+  # codingCutoff parameter of isoformSwitchAnalyzer, see
+  # https://rdrr.io/bioc/IsoformSwitchAnalyzeR/man/analyzeCPAT.html
+  coding_cutoff: 0.725
+  # Should be set to true when using de-novo assembled transcripts.
+  remove_noncoding_orfs: false
```

Comment:
Need to add the alpha/sig-level here. I would like to harmonize the notation for significance in the pipeline. Maybe even just one global threshold? Allowing to set it differently everywhere seems both error prone and even problematic from a philosophic perspective.

Reply:
That sounds good!